Update documentation of autotailor (#2042)
OpenSCAP/openscap: docs/manual/manual.adoc

The downloaded guidance contains rule descriptions, but it doesn't contain OVAL checks which could be used for evaluation by OpenSCAP. You can find guidances with implemented OVAL checks and also with remediations at https://github.com/ComplianceAsCode/content[ComplianceAsCode] project, which contains wide range of profiles.

*How can I create a tailoring file if I can't install SCAP Workbench?*

> Use the `autotailor` tool which allows you to create tailoring files on command line.

Reviewer: ```suggestion
Use the `autotailor` tool which allows you to create tailoring files using command line options.
```

For more information, please refer to section <<_tailoring,Tailoring>>.

*I try to apply a tailoring file, but OpenSCAP still evaluates rules that I have unselected. How can I enforce my changes of the profile?*

Make sure that you provide the ID of the customized profile in `--profile` option instead of the ID of the original profile.
If you created the tailoring file using `autotailor` and you haven't used the `-p` or `--new-profile-id` option, the ID of the customized profile is the original profile ID with `_customized` suffix appended.
If you created the tailoring file using SCAP Workbench, you were prompted to choose the ID of the customized profile. By default, the ID of the customized profile is the original profile ID with `_customized` suffix appended.

> You can display the ID of the customized profile by running `oscap info <your_tailoring_file>`.

Reviewer: ```suggestion
You can find the ID of the customized profile with `oscap info <your_tailoring_file>` command.
```
